src/components/tooltip/Marker.js

```diff
@@ -235,6 +235,9 @@ function getMarkerDetails(
           <TooltipDetails>
             <TooltipDetail label="Operation">{data.operation}</TooltipDetail>
             <TooltipDetail label="Source">{data.source}</TooltipDetail>
+            {data.threadId ? (
```

Comment:
nit: our practice in this project is to always (or let's say mostly always) compare values with something. In this case compare with undefined, right? From the cpp code, I believe it won't be null if present.

Reply:
Yes, it's non-null and can be undefined only.